packs/data/spell-effects.db/spell-effect-wish-twisted-form-critical-failure.json

```diff
@@ -0,0 +1,99 @@
+{
+    "_id": "f5EOSVWSkG8RmXee",
+    "img": "systems/pf2e/icons/spells/wish-twisted-form.webp",
+    "name": "Spell Effect: Wish-Twisted Form (Critical Failure)",
```

Comment:
Since failure and critical failure have the same duration we can condense them down too with another ChoiceSet.

Reply:
Would it be appropriate to just remove the on-success effect then? I only added it since I was already splitting up failure and critical failure. Similar spells like Hyperfocus only have a single effect. Assuming there's no way to modify duration with a Rules Element.

Reply:
There is currently no way to alter the duration with a rule element. So we can keep the success version and just condense it later when we get that ability.